package.json

```diff
@@ -15,6 +15,7 @@
     },
     "dependencies": {
         "@google-cloud/functions-framework": "^3.3.0",
+        "@react-oauth/google": "^0.12.1",
```

Comment:
It seems additional to have this when firebase/auth has functions that covers the features from this library.